chain/app/app.go

```diff
@@ -31,18 +32,18 @@ import (
 )
 
 const (
-	appName          = "band"
+	appName          = "BandApp"
```

Comment:
Not relevant to the "Upgrade to be compatible with gaia 2.0.4"
But it's ok.

Reply:
True